### ISSUE TITLE
Remove factory_girl from production gemfile

### ISSUE THREAD

**Issue by bess** [curationexperts/laevigata#367]
factory_girl should not be in our production gemfile once we're running a real production service, but we want it there while we're still in development because it is very helpful to be able to build fixture objects on our production server to test and demo.

**Comment by bess** [comment]
Note: This needs to happen just before we go live, but not until AFTER we do the training classes. We will need factory_girl in order to generate data for those classes.

**Comment by bess** [comment]
Won't fix for now. It's too useful to be able to generate test data.
